priv/templates/boot_loader.eex

```diff
@@ -16,7 +16,7 @@ _cleanup() {
     trap - HUP INT TERM QUIT EXIT USR1 USR2
     echo ""
     echo "Shutting down.."
-    "$RELEASES_DIR/$REL_VSN/$REL_NAME.sh" rpc init stop
+    "$RELEASES_DIR/$REL_VSN/$REL_NAME.sh" <%= signal_handler %>
     wait $__pid
```

Comment:
imo it's fine if the user of distillery wants to do something potentially stupid like block in a signal handler forever. If the process really needs to be killed, a sigkill can be sent.

We have an situation where we're running elixir in docker containers on mesos (sigh....) and mesos will impose a time window where the container can exit and if it doesn't exit in that window it's forcibly killed. So that's the context where the request for this functionality came from - there's a orchestrator that can kill the erlang vm if it decides it needs to (for better or for worse).

I completely understand if you think this feature will be more painful than useful due to folks using it incorrectly, in which case we can continue using our forked version until OTP 20 when all of this becomes irrelevant :100: